aidb/inference/bound_inference_service.py

```diff
@@ -226,6 +229,40 @@ async def infer(self, inputs: pd.DataFrame):
       await self._insert_in_cache_table(inputs_not_in_cache_primary_cols, conn)
       await self._insert_output_results_in_tables(records_to_insert_in_table, inputs_not_in_cache, conn)
 
+    if if_return:
+      return_res = records_to_insert_in_table.copy()
+
+      # Retrieve cached results
+      tuples_for_condition = []
```

Comment:
what does `tuples_for_condition` mean?

Reply:
Store the keys associated with each sampled blob. Rename it to 'sampled_key_list'